meta.json

```diff
@@ -227,7 +215,37 @@
                             "title": "Image overlay",
                             "shortDescription": "Display an animated weather radar",
                             "published": true
-                        }
+                        },
+                        {
+                            "uid": "extruded-objects",
+                            "title": "Extruded geoshapes",
+                            "shortDescription": "Display a map with the 3D extruded object",
+                            "published": true
+                        },
+                        {
+                            "uid": "resizable-polyline",
+                            "title": "Resizable Polyline",
+                            "shortDescription": "Display resizable polyline on a map",
+                            "published": true
+                        },
+                        {
+                            "uid": "resizable-polygon",
+                            "title": "Resizable Polygon",
+                            "shortDescription": "Display resizable polygon on a map",
+                            "published": true
+                        },
+                        {
+                            "uid": "resizable-circle",
+                            "title": "Resizable Circle",
+                            "shortDescription": "Display resizable circle on a map",
+                            "published": true
+                        },
+                        {
+                            "uid": "resizable-rect",
+                            "title": "Resizable Rectangle",
+                            "shortDescription": "Display resizable rectangle on a map",
+                            "published": true
+                        },
```

Comment:
Shall we introduce just a new subcategory on the developer.here.com something like "Resizable Objects"?